guacamole/src/main/webapp/translations/de.json

```diff
@@ -223,6 +223,7 @@
         "FIELD_HEADER_TYPE"     : "Typ:",
 
         "NAME_TYPE_BALANCING"       : "Balancing",
+        "NAME_TYPE_ROUNDROBIN"      : "RoundRobin",
```

Comment:
FYI - you do not need to define translation strings for languages other than English if the string does not differ from English. The translation system will pull the English strings as a fallback.

If you speak the language in question, and it just happens to be the same as English in this case, then that's a different matter, but otherwise - it's better to leave it blank. Leaving it blank allows others to see that the string has not been defined for that language yet, and to then provide a real translation.